### PR TITLE
Added r-rbf CRAN package

### PR DESCRIPTION
**Pull request by khughitt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-rbf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('RBF')"           # [not win]
+    - "\"%R%\" -e \"library('RBF')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=RBF
+  license: GPL (>= 3.0)
```
Comment (dbast) [inline recipes/r-rbf/meta.yaml:47]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/r-rbf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  summary: A robust backfitting algorithm for additive models based on (robust) local polynomial
+    kernel smoothers. It includes both bounded and re-descending (kernel) M-estimators,
+    and it computes predictions for points outside the training set if desired. See
+    Boente, Martinez and Salibian-Barrera (2017) <doi:10.1080/10485252.2017.1369077>
+    for details.
+
+  license_family: {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```
Comment (dbast) [inline recipes/r-rbf/meta.yaml:54]:
```suggestion
  license_file: {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

**Comment by khughitt** [comment]
Good catches!

**Comment by dbast** [comment]
@khughitt Thanks

**Pull request merged**